run/hello-broken/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "hello-broken",
+  "private": true,
+  "description": "Broken Cloud Run service for troubleshooting practice",
+  "main": "index.js",
+  "scripts": {
+    "start": "node index.js",
+    "test": "echo \"Error: no test specified\" && exit 0",
+    "e2e-test": "TARGET=Cloud test/runner.sh mocha test/system.test.js --timeout=20000"
```

Comment:
It might be worth it to just put this script as `test` so you may [`npm test`](https://docs.npmjs.com/cli/test.html) instead of `npm run e2e-test`.

Reply:
Out of scope of this PR.

Reply:
Actually, now that I see your point about `npm test` being a no-op, this bears some follow-up. I will file an issue.

Reply:
I've looked at the other samples that include script `e2e-test` in this repo as well as the cDPE test guide and squad notes.

If we consider this change out of scope, then I oppose adding this line then:

```js
"test": "echo \"Error: no test specified\" && exit 0",
```

We do specify a test in this repo. We don't specify a config for specifying a test as the error suggests.

If we're going to add this script:

```js
"e2e-test": ...
```

We should either consider using the Node way of `npm test` or fix the error message for `npm test` to be more accurate.

Reply:
#1493